apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,74 +9,86 @@ group: core
+The borderless expandable block is useful in tighter areas where using a default expandable block would visibly cause too many blocks. The borderless expandable block is achieved by using the `styleType` prop.
 
-### Expandable blocks
+<LiveExample src='ExpandableBlock.borderless.tsx'>
+  <AllExamples.ExpandableBlockBorderlessExample client:load />
+</LiveExample>
+
+## Usage
+
+Expandable blocks are used to progressively disclose information. [Read this article to learn more about progressive disclosure.](https://www.nngroup.com/articles/progressive-disclosure/) Expandable blocks should not be used with flattened hierarchy or information panels.
+
+### Forms
 
 Expandable blocks are used in pages to divide up content in sections, in forms for example. If each section has many fields, using expandable blocks will help alleviate the UI and prevent the user from scrolling endlessly to get to the section they are looking for.
 
-Expanded block containing form elements within an interface.
+<LiveExample src='ExpandableBlock.form.tsx'>
+  <AllExamples.ExpandableBlockFormExample client:load />
+</LiveExample>
 
-### Expandable blocks VS hierarchy tree
+### Expandable blocks vs hierarchy tree
 
-While these two patterns may appear similar, they both fulfill a different role and are not interchangeable. Here are some tips to determine if an expandable block or row is what you need.
+While these two patterns may appear similar, they both fulfill a different role and are not interchangeable. Here are some tips to determine if an expandable block is what you need.
 
-Use an expandable rows/blocks when:
+Use an expandable block when:
 
-- You need a row in a table to display additional metadata that doesn’t align with the column content;
-- You have a page with different sections all containing user-generated content that would benefit from being split up;
-- There is only one level of indentation possible at all times;
-- The content you want to be expandable varies from simple list items to more complex form components (such as text fields, buttons, etc.).
+- You have a page with different sections all containing user-generated content that would benefit from being split up
+- There is only one level of indentation possible at all times
+- The content you want to be expandable varies from simple list items to more complex form components (such as text fields, buttons, etc.)
 
-Use a hierarchy tree when:
+Use a [hierarchy tree](tree) when:
 
-- You require a pattern that works with several levels of data (folder drilling);
-- There are only list items in the pattern;
-- The hierarchy is not necessarily linear and may branch out.
+- You require a pattern that works with several levels of data (folder drilling)
+- There are only list items in the pattern
+- The hierarchy is not necessarily linear and may branch out
 
 ## Props
 
 <PropsTable path={frontmatter.propsPath} />
```

Comment:
Fairly new feature, but you can include Related Components section.

https://github.com/iTwin/iTwinUI/blob/55147212471f644cd352f43c9e79e823f5608ac9/apps/website/src/pages/docs/input.mdx?plain=1#L75

Reply:
Is there any other related components that need to go here outside of Tree? Or are we putting any mentions of other components in this section?

Reply:
In the old documentation we wrote about expandable rows within a table.  Maybe somewhere in the new docs we can write something like "Table rows can also expand similarly to expandable block" with a link to table & then we could link table as a related component.

Reply:
Added